src/Type/Php/StrtotimeFunctionReturnTypeExtension.php

```diff
@@ -30,20 +31,27 @@ public function isFunctionSupported(FunctionReflection $functionReflection): boo
 	public function getTypeFromFunctionCall(FunctionReflection $functionReflection, FuncCall $functionCall, Scope $scope): Type
 	{
 		$defaultReturnType = ParametersAcceptorSelector::selectSingle($functionReflection->getVariants())->getReturnType();
-		if (count($functionCall->getArgs()) === 0) {
+		if (count($functionCall->getArgs()) !== 1) { // strtotime() & 2nd param baseTimestamp are both unsupported use cases
 			return $defaultReturnType;
 		}
 		$argType = $scope->getType($functionCall->getArgs()[0]->value);
 		if ($argType instanceof MixedType) {
 			return TypeUtils::toBenevolentUnion($defaultReturnType);
 		}
-		$result = array_unique(array_map(static fn (ConstantStringType $string): bool => is_int(strtotime($string->getValue())), TypeUtils::getConstantStrings($argType)));
+		$results = array_unique(array_map(static fn (ConstantStringType $string): int|bool => strtotime($string->getValue()), TypeUtils::getConstantStrings($argType)));
+		$resultTypes = array_unique(array_map(static fn (int|bool $value): string => gettype($value), $results));
 
-		if (count($result) !== 1) {
+		if (count($resultTypes) !== 1 || count($results) === 0) {
 			return $defaultReturnType;
 		}
 
-		return $result[0] ? new IntegerType() : new ConstantBooleanType(false);
+		if ($results[0] === false) {
+			return new ConstantBooleanType(false);
+		}
+
+		$results = array_map('intval', $results);
+
+		return IntegerRangeType::createAllGreaterThan(min($results));
```

Comment:
This yields:

  57     Parameter #1 $value of static method                           
         PHPStan\Type\IntegerRangeType::createAllGreaterThan() expects  
         float|int, int|false given.    

But I don't get it, because $results is not empty, and so min() cannot return false.